examples/persistence-service-catalog.md

```diff
@@ -44,43 +43,33 @@ cd /Temp/my-spring-boot
 sd create -t crud -i my-spring-boot
```

Comment:
This command should be converted to a curl command + unzip, otherwise `sd` instructions to install it will be required

Reply:
Good idea, I will update